angry-bot/src/data/tarot-texts.ts

```diff
@@ -564,33 +564,33 @@ export const tarots: Tarot[] = [
         text: "Up until the 19th century, people used to be :angry all the time. But then they discovered that being angry all the time is good for your health. So they stopped being angry. And that is why not everybody is angry today. Pathetic.",
```

Comment:
We can mabe fix this `:angry` aswell or is it intended? :o

Reply:
or should that be `:angry191:`?

Reply:
is not intended! Thanks!

Reply:
fixed it in new push!